utils/findLinks.js

```diff
@@ -0,0 +1,64 @@
+
+const chalk = require('chalk')
+
+
+const findLinks = (datos, urlF) => {
+    let allthelink = /\[(.*)\]\S((http|https):\/\/[^\s\n)]+)(?=\))/gim;
+    let regularL = /(http|https):\/\/.*\)?/gi;
+    let textReg = /\[(.*)\]/gim
+    let objectLinks = [];
+    let newFile = process.argv.indexOf('--file');
+    let uri = process.argv[newFile + 1];
+    let joinArray = []
+    let linkAndtext;
+    let joinObject;
+    return new Promise((resolve) => {
+        let noStr = datos;
+        let str = noStr.toString();
+        let myArray;
```

Comment:
El nombre de las variables no es claro el ¿para que se usaran?
```suggestion
        let nameStatistics = data;
        let statistics = nameStatistics.toString();
        let arrayStatistics;
```